src/index.js

```diff
@@ -188,7 +188,16 @@ class Fastly {
     return this.request.get(`/service/${this.service_id}/wafs/${wafId}/rule_statuses?filter[status]=${wafStatus}&page[size]=200&page[number]=${pageNumber}`)
   }
 
-
+  /**
+   * Gets the WAF Rules associated with a service dependant on WAF tags.
+   * @param wafId {string} The WAF ID associated with a service.
+   * @param tag {string} The WAF tag whose rules are enabled on a service.
+   * @return {Promise} The response object representing the completion or failure.
+   * @param pageNumber {number} Page number for the of the results output.
```

Comment:
Might be pedantic, but all the other methods in this file has the `@param` listed then the `@return` statement last.

Reply:
oh damn it. Forgot that one - good spot ! Will add that.